packages/web-components/fast-components/src/listbox-option/listbox-option.styles.ts

```diff
@@ -35,7 +38,15 @@ import {
     typeRampBaseFontSize,
     typeRampBaseLineHeight,
 } from "../design-tokens";
-import { heightNumber } from "../styles/size";
+
+export const optionHeight = DesignToken.create<number>("option-height").withDefault(
```

Comment:
This model is changing soon. It will be padding based instead. If that still works here that's fine and we can adjust this with the rest of that change. If there is anything relying on calculating based on this height, that won't work longer term. I think that might be what `maxHeightChanged` above is doing though.

Reply:
We're not holding for density in this. We need to ship multiselect.